bitnami/pytorch/README.md

```diff
@@ -0,0 +1,161 @@
+| `image.debug`                          | Specify if debug logs should be enabled                                                      | `false`                                                 |
+| `git.registry`                         | Git image registry                                                                           | `docker.io`                                             |
+| `git.repository`                       | Git image name                                                                               | `bitnami/git`                                           |
+| `git.tag`                              | Git image tag                                                                                | `latest`                                                |
+| `git.pullPolicy`                       | Git image pull policy                                                                        | `Always`                                                |
+| `git.pullSecrets`                      | Specify docker-registry secret names as an array                                             | `[]` (does not add image pull secrets to deployed pods) |
+| `pytorch.entrypoint.file`              | Main entrypoint to your application                                                          | `main.py`                                               |
+| `pytorch.entrypoint.args`              | Args required by your entrypoint                                                             | `nil`                                                   |
+| `pytorch.distributed.enabled`          | Enable distributed mode for PyTorch                                                          | `true`                                                  |
+| `pytorch.distributed.worldSize`        | Number of nodes that will execute your code                                                  | `4`                                                     |
+| `pytorch.configMap`                    | Config map that contains the files you want to load in PyTorch                               | `nil`                                                   |
+| `pytorch.cloneFilesFromGit.enabled`    | Enable in order to download files from git repository                                        | `false`                                                 |
+| `pytorch.cloneFilesFromGit.repository` | Repository that holds the files                                                              | `nil`                                                   |
+| `pytorch.cloneFilesFromGit.revision`   | Revision from the repository to checkout                                                     | `master`                                                |
+| `pytorch.extraEnvVars`                 | Extra environment variables to add to master and workers pods                                | `nil`                                                   |
```

Comment:
I don't see why it's necessary to create a "pytorch" object? I mean, why is it necessary to have this structure:

```yaml
pytorch:
  distributed:
    enabled: true
    worldSize: ...
  configMap: ...
  cloneFilesFromGit:
    enabled: true
    repository: ...
    revision: ...
  extraEnvVars: ...
```

I'd use:

```yaml
mode: standalone (or distributed)
worldSize: ... # ignored if mode=standalone
configMap: ...
gitClone:
  enabled: true
  repository: ...
  revision: ...
extraEnvVars: ...
```